dashboards/frontend-application.json

```diff
@@ -9,7 +9,7 @@
       "pluginName": "Loki"
```

Comment:
Can you provide before/after screenshots of the dashboard?

Reply:
Here you go:
![events_app_dashboard](https://user-images.githubusercontent.com/847684/187438559-4b473049-fd20-4c25-9c99-312097f47920.png)

Reply:
And for the receiver one, event count got added:
![events_receiver_dashboard](https://user-images.githubusercontent.com/847684/187438873-9232b5e5-98df-4bf9-b28d-44eecb5fef21.png)